Client/Ecosia/Analytics/Analytics.swift

```diff
@@ -89,6 +89,14 @@ final class Analytics: AnalyticsProtocol {
             .property(OnboardingCardNTPExperiment.analyticsProperty))
     }
     
+    func ntpSeedCounterExperiment(_ action: Action, seedCounterManager: SeedProgressManagerProtocol.Type) {
+        track(Structured(category: Category.climateCounter.rawValue,
+                         action: action.rawValue)
+            .label(Label.Browser.newTab.rawValue)
```

Comment:
I think for NTP things we usually use `Category.ntp` instead of label (seems `Label.Browser.newTab` is not even used anywhere else 🤷). Would that make more sense to keep in line with other events? We could then have `climate_counter` as the label instead.